util/connector.py

```diff
@@ -0,0 +1,16 @@
+import os
+import requests
+
+
+MODEL_PROVIDER_BASE_URI = os.getenv("MODEL_PROVIDER")
+
+
+def get_model_path(project_id: str, model_name: str, revision: str = None) -> str:
+    url = f"{MODEL_PROVIDER_BASE_URI}/model_path"
+    params = {
+        "project_id": project_id,
+        "model_name": model_name,
+        "revision": revision,
```

Comment:
will be changed i guess (only model name)

Reply:
changed